### PR TITLE
fix(cart): importing magento driver module breaks app builds

### PR DESCRIPTION
**Pull request by griest024**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Importing `DaffCartMagentoDriverModule` and using it in an app will break the build with error:
```
Error during template compile of 'MagentoModule'
  Function calls are not supported in decorators but 'DaffCartMagentoDriverModule' was called.
```

## What is the new behavior?
The app using the module will correctly build.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

**Comment by damienwebdev** [comment]
@griest024 In theory, this should be breaking the demo app. Can you determine why it isn't? I'd like to be able to verify this behavior and prevent future regressions.

**Comment by griest024** [comment]
@damienwebdev I don't think that demo is even importing the Magento module.

**Comment by damienwebdev** [comment]
@griest024 can we import this into demo (even if we don't use the driver) and validate that this bug is in-fact fixed?

File: apps/demo/src/app/app.module.ts [hunk 1]
```diff
@@ -38,16 +39,17 @@ import { DemoCartRootModule } from './cart/cart-root.module';
 
     HttpClientModule,
 
-    InMemoryModule,
-    
+    // InMemoryModule,
```
Comment (damienwebdev) [inline apps/demo/src/app/app.module.ts]: I don't think we should make any changes in this file.

Comment (griest024) [reply apps/demo/src/app/app.module.ts]: I thought it was necessary to break the demo but I was wrong. Changed in [`284a5d1`](https://github.com/graycoreio/daffodil/pull/691/commits/284a5d18ce9adaf28c04a15a92ff7d9cd9556ee9).

**Pull request merged**